nosqlbench/http-jsonapi-search-filter-sort.yaml

```diff
@@ -1,6 +1,6 @@
 min_version: "5.17.3"
```

Comment:
same here:

```
$ nb5 -v --report-csv-to logs --log-histograms logs/hdr_metrics.log http-jsonapi-search-filter-sort jsonapi_host=localhost docscount=10000 connections=25 rate=250
     760 INFO  [main] NBCLI        Configured scenario log at logs/scenario_20230502_132401_299.log
     767 INFO  [main] NBCLI        Running NoSQLBench Version 5.17.3-release
     768 INFO  [main] NBCLI        command-line: -v --report-csv-to logs --log-histograms logs/hdr_metrics.log http-jsonapi-search-filter-sort jsonapi_host=localhost docscount=10000 connections=25 rate=250
    2653 INFO  [main] NBCLI        client-hardware: {"memory-GiB":"31.10","processor-name":"Intel(R) Core(TM) i7-10750H CPU @ 2.60GHz","heap-max-GiB":"7.78","if-speeds":["0Mib","10Gib"],"physical-cores":"6","max-frequency-ghz":"0.01","logical-cores":"12","sockets":"1"}
    2845 INFO  [main] OpsLoader    workload URI: 'file:///home/ise/workspace/datastax/stargate/jsonapi/nosqlbench/http-jsonapi-search-filter-sort.yaml'
    2910 ERROR [main] ERRORHANDLER internal error: class java.util.LinkedHashMap cannot be cast to class java.lang.String (java.util.LinkedHashMap and java.lang.String are in module java.base of loader 'bootstrap')
java.lang.ClassCastException: class java.util.LinkedHashMap cannot be cast to class java.lang.String (java.util.LinkedHashMap and java.lang.String are in module java.base of loader 'bootstrap')
	at io.nosqlbench.engine.api.scenarios.NBCLIScenarioParser.parseScenarioCommand(NBCLIScenarioParser.java:157)
	at io.nosqlbench.engine.cli.NBCLICommandParser.parse(NBCLICommandParser.java:87)
	at io.nosqlbench.engine.cli.NBCLIOptions.parseAllOptions(NBCLIOptions.java:631)
	at io.nosqlbench.engine.cli.NBCLIOptions.<init>(NBCLIOptions.java:245)
	at io.nosqlbench.engine.cli.NBCLIOptions.<init>(NBCLIOptions.java:236)
	at io.nosqlbench.engine.cli.NBCLI.applyDirect(NBCLI.java:248)
	at io.nosqlbench.engine.cli.NBCLI.apply(NBCLI.java:114)
	at io.nosqlbench.engine.cli.NBCLI.main(NBCLI.java:94)
Scenario stopped due to error. See logs for details.

```

Reply:
Fixed. The workload will now run to completion but will indicate errors when using behavior not yet implemented.